.travis.yml

```diff
@@ -10,7 +10,9 @@ services:
   - docker
 
 before_install:
+  - ./tools/travis/setupscan.sh
```

Comment:
I agree that the right call is to stay consistent with the rest of the repos. 

When I picked on `.travis` I was looking to isolate the code from the CI parts, similarly with `.git` folder. It's not such a big difference worth going for at this point when the other repos don't follow this convention.